backend/src/v2/compiler/argocompiler/argo.go

```diff
@@ -41,8 +42,13 @@ type Options struct {
 	// optional
 	PipelineRoot string
 	// TODO(Bobgy): add an option -- dev mode, ImagePullPolicy should only be Always in dev mode.
+	TtlSeconds int32
 }
 
+const (
+	pipelineDefaultTtlSeconds = int32(30)
```

Comment:
Just checking... In the PR's description, you mention `60s`. Is `30` really intended here?

Reply:
Yes, I changed the description accordingly.